packages/modal-ui-js/src/lib/render-modal.ts

```diff
@@ -19,6 +19,7 @@ import { translate } from "@near-wallet-selector/core";
 export type HardwareWalletAccountState = HardwareWalletAccount & {
   selected: boolean;
 };
+let renderCount = 0;
```

Comment:
Could it be better if we make it `let initialRender = true` instead , `let renderCount = 0; `and later switch to false instead incrementing?

Reply:
Yes, this is just a workaround at the moment since we want to include this in this release. I can set it as you suggested.